packages/pigment-css-react/HOW_PIGMENT_CSS_WORKS.md

```diff
@@ -0,0 +1,151 @@
+# How Pigment CSS works
+
+Pigment CSS is a zero-runtime CSS-in-JS library. This means it does not have access to the end user's browser runtime which would be necessary to generate and insert authored CSS at runtime. Instead, it does all its processing at build time to pre-generate the CSS which then becomes part of the output bundle. This is why it cannot be consumed on its own. You must install the Pigment CSS plugin that corresponds to the framework you're using and configure your bundler accordingly—Pigment CSS currently supports Next.js and Vite.
+
+- [Processor](#processor)
+  - [1. Detection and evaluation](#1-detection-and-evaluation)
+  - [2. Transformation and runtime replacement](#2-transformation-and-runtime-replacement)
+  - [3. Extraction](#3-extraction)
+- [Debugging](#debugging)
+
+## Processor
+
+Pigment CSS uses the [WyW-in-JS](https://wyw-in-js.dev/) library that also powers [Linaria](https://linaria.dev/). It features a [processor](https://wyw-in-js.dev/how-to/custom-tagged-template#creating-a-processor) which makes it possible to create custom logic that's triggered by the presence of different imports from the library. The processor looks through the source code for `styled()`, `css()`, and other function calls and extracts the arguments to be evaluated. These evaluated values are then handed back to Pigment CSS for additional customization on top.
```

Comment:
There are two different types of references here. One is where we tell about the import of `styled`/`css`. In that case, it would be the same.
But in any other case, it would be a function call. So it should be `styled()`/`css()`.

Reply:
Updated.